seed/views/v3/labels.py

```diff
@@ -61,20 +107,8 @@ def get_queryset(self):
 
     def get_serializer(self, *args, **kwargs):
         kwargs['super_organization'] = self.get_organization(self.request)
-        inventory = InventoryFilterBackendWithInvType().filter_queryset_with_inv(
+        inventory = filter_labels_for_inv_type(
             request=self.request, inv_type=None
```

Comment:
What do you think about just defaulting inv_type to None so you don't have to provide it here?